docs/dev/existing_data_updates.rst

```diff
@@ -31,19 +39,20 @@ archivers themselves.
 **1.2)** Update the dictionary of production DOIs in :mod:`pudl.workspace.datastore` to
 refer to the new raw input archives.
 
-**1.3)** In :py:const:`pudl.metadata.sources.SOURCES`, update the ``working_partitions``
-to reflect the years of data that are available within each dataset and the
-``records_liberated`` to show how many records are available. Check to make sure other
-fields such as ``source_format`` or ``path`` are still accurate.
+**1.3)** If you're adding a new year, in :py:const:`pudl.metadata.sources.SOURCES`,
```

Comment:
"adding a new year" could be "increasing coverage" or something that is more inclusive of non-year partitions

Reply:
That's true...the reason I used "year" here specifically is because sometimes (as in the case for 923 quarterly updates) when we add the monthly data we don't update the year partitions because the new data is within the same year. So this step is unnecessary.